doc/releases/quincy.rst

```diff
@@ -0,0 +1,3656 @@
+
+   #. Wait for the cluster to deactivate any non-zero ranks by
+      periodically checking the status::
+
+	# ceph status
+
+   #. Take all standby MDS daemons offline on the appropriate hosts with::
+
+	# systemctl stop ceph-mds@<daemon_name>
+
+   #. Confirm that only one MDS is online and is rank 0 for your FS::
+
+	# ceph status
+
+   #. Upgrade the last remaining MDS daemon by installing the new
+      packages and restarting the daemon::
+
+        # systemctl restart ceph-mds.target
+
+   #. Restart all standby MDS daemons that were taken offline::
+
+	# systemctl start ceph-mds.target
+
+   #. Restore the original value of ``max_mds`` for the volume::
+
+	# ceph fs set <fs_name> max_mds <original_max_mds>
+
+#. Upgrade all radosgw daemons by upgrading packages and restarting
+   daemons on all hosts::
+
+     # systemctl restart ceph-radosgw.target
+
+#. Complete the upgrade by disallowing pre-Quincy OSDs and enabling
+   all new Quincy-only functionality::
+
+     # ceph osd require-osd-release quincy
+
+#. If you set ``noout`` at the beginning, be sure to clear it with::
+
+     # ceph osd unset noout
+
+#. Consider transitioning your cluster to use the cephadm deployment
+   and orchestration framework to simplify cluster management and
+   future upgrades.  For more information on converting an existing
+   cluster to cephadm, see :ref:`cephadm-adoption`.
+
+Post-upgrade
+~~~~~~~~~~~~
+
+#. Verify the cluster is healthy with ``ceph health``.
+
+#. If you are upgrading from Mimic, or did not already do so when you
+   upgraded to Nautlius, we recommened you enable the new :ref:`v2
+   network protocol <msgr2>`, issue the following command::
+
+     ceph mon enable-msgr2
+
+   This will instruct all monitors that bind to the old default port
+   6789 for the legacy v1 protocol to also bind to the new 3300 v2
+   protocol port.  To see if all monitors have been updated,::
+
+     ceph mon dump
+
+   and verify that each monitor has both a ``v2:`` and ``v1:`` address
+   listed.
+
+#. Consider enabling the :ref:`telemetry module <telemetry>` to send
+   anonymized usage statistics and crash information to the Ceph
+   upstream developers.  To see what would be reported (without actually
+   sending any information to anyone),::
+
+     ceph telemetry preview-all
+
+   If you are comfortable with the data that is reported, you can opt-in to
+   automatically report the high-level cluster metadata with::
+
+     ceph telemetry on
+
+   The public dashboard that aggregates Ceph telemetry can be found at
+   `https://telemetry-public.ceph.com/ <https://telemetry-public.ceph.com/>`_.
+
+   For more information about the telemetry module, see :ref:`the
+   documentation <telemetry>`.
+
+
+Upgrading from pre-Octopus releases (like Nautilus)
+---------------------------------------------------
+
+
+You *must* first upgrade to Octopus (15.2.z) or Pacific (16.2.z) before
+upgrading to Quincy.
+
+
+Changelog
+---------
```

Comment:
Should changelog be regenerated to pick up the most recent quincy branch merges?

Reply:
No.  I figured out there shouldn't actually be a changelog for a first release.